docs/scenarios_list.md

```diff
@@ -411,6 +411,12 @@ nav_order: 3
 * Check if list with two organizations is returned in case 'report' table contains reports for two organizations
 * Check if list with two organizations is returned in case 'report' table contains multiple reports for two organizations
 
+## `insights-results-aggregator/results_for_cluster_list.feature`
```

Comment:
I couldn't find this feature in https://github.com/RedHatInsights/insights-behavioral-spec/tree/main/features/insights-results-aggregator. is it missing?